CONTRIBUTING.md

```diff
@@ -30,6 +30,34 @@ works.
 
 This document will guide you through the contribution process.
 
+### Rules of thumb
+
+1. #### Provide motivation for the change
+   Why will this change make the code better; does it fix a bug, is it a new
+   feature, etc. This should be in the commit messages as well as in the PR
+   description.
+2. #### Don't make _unnecessary_ code changes
+   Things that are changed because of personal preferance or style, like:
+   renaming of variables or functions, adding or removing white spaces,
+   reordering lines or whole code blocks. These sort of changes should have
+   a good reason since they cause unnecessary ["code churn"](https://blog.gitprime.com/why-code-churn-matters).
+   As part of the projects strategy we maintain several lines of code, code 
```

Comment:
I believe something like "we maintain multiple release lines" would be more clear?